ministryofjustice/correspondence_tool_staff
# CT-2364 Introduce Offender SAR Monthly report download option

## Description
Allows a user to download a monthly KPI report based on existing SAR/FOI monthly reports. Does not include any bespoke amendments for Offender SARs (i.e. initial plumbing & download functionality only).

## Notes
- Rules for `business_unit_in_time?` will require consideration
- Report output uses existing `trigger` terminology, reconsider
- User restrictions not implemented
 
## Self-review checklist

 
### Screenshots
<img width="850" alt="Screen Shot 2019-09-04 at 11 02 31" src="https://user-images.githubusercontent.com/47113046/64245917-c183c280-cf03-11e9-9a61-e3e2d3f43a18.png">

 
### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2364
 
### Deployment
1. Database migration required
2. Data Migration required
 
### Manual testing instructions
1. Log in as any user
2. Performance -> Offender Sars section. 
3. Select 'download' should download an Excel file
